.fixtures.yml

```diff
@@ -5,6 +5,6 @@ fixtures:
       ref: '1.0.0'
     stdlib:
       repo: "git://github.com/puppetlabs/puppetlabs-stdlib.git"
-      ref: '2.4.0'
+      ref: '2.5.1'
```

Comment:
you're only bumping this here, not in manifest.json

also, with such a major rewrite, we could be more daring ;)
(but that's just my *opinion*)